introduction.md

```diff
@@ -1,2 +1,9 @@
 Hello, this is me!
-Ruby is my favorite.
\ No newline at end of file
+Ruby is my favorite.
+
+Deadly Skills
+-------------
+* Ruby
+* Git
+* Perl
+* Node.js
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?